packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -259,6 +262,37 @@ export function TransferPanelSummary({ token }: TransferPanelSummaryProps) {
           )}
         </span>
       </div>
+      {(isDestinationChainArbitrumOne || isDestinationChainArbitrumSepolia) && (
+        <div
+          className={twMerge(
+            'grid grid-cols-[260px_auto] items-center text-sm font-light'
+          )}
+        >
+          <ConfirmationTimeInfo chainId={networks.sourceChain.id} />
+        </div>
+      )}
     </TransferPanelSummaryContainer>
   )
 }
+
+function ConfirmationTimeInfo({ chainId }: { chainId: number }) {
+  const { confirmationTimeInReadableFormat, isDefaultConfirmationTime } =
+    getConfirmationTime(chainId)
+  return (
+    <>
+      <span>Confirmation time:</span>
+      <span className="flex items-center font-medium">
+        {confirmationTimeInReadableFormat}
+        {!isDefaultConfirmationTime && (
+          <Tooltip
+            content={
+              'Fast Withdrawals relies on a committee of validators. In the event of a committee outage, your withdrawal falls back to the 7 day challenge period secured by Arbitrum Fraud Proofs.'
+            }
+          >
+            <InformationCircleIcon className="ml-1 h-3 w-3" />
+          </Tooltip>
+        )}
+      </span>
```

Comment:
no big deal but it looks a bit squashed on mobile
<img width="442" alt="image" src="https://github.com/user-attachments/assets/8f4f3208-ee23-49d2-ac6e-530a7139ee03">